types/node/tslint.json

```diff
@@ -21,6 +21,7 @@
     "prefer-const": false,
     "prefer-method-signature": false,
     "strict-export-declare-modifiers": false,
-    "unified-signatures": false
+    "unified-signatures": false,
+    "void-return": false
```

Comment:
Why is it needed to disable the `void-return` lint warning?

Reply:
The typings are full of `void` and `Promise<void>` return types, I honestly don't know how it passed until now

Reply:
This is a false positive. Void is nowhere used as a param and only as a return type which is the kind of behavior the [rule](https://github.com/Microsoft/dtslint/blob/master/docs/void-return.md) enforces